lib/metrics.js

```diff
@@ -1,33 +1,33 @@
-'use strict';
-var util = require('util');
+'use strict'
+const util = require('util')
 
 //
 // --- Metric (base class)
 //
 
 function Metric(key, tags, host) {
-    this.key = key;
-    this.tags = tags || [];
-    this.host = host || '';
+    this.key = key
```

Comment:
I don't love the removal of the semicolons here—feels unnecessary and makes it harder to tell what significant code changes this PR is making

Reply:
I'll add them back, don't have a strong opinion.

Reply:
I updated the PR and removed all styling changes.